src/Model/ConfigGroupI18n.php

```diff
@@ -58,7 +58,7 @@ public function getDescription(): string
      * @param string $name Config group name
      * @return ConfigGroupI18n
      */
-    public function setName(string $name): string
+    public function setName(string $name): AbstractI18n
```

Comment:
Please change return type to self and use English commit messages. And if you could change also return type to in setDescription couple lines above.